codegen/smithy-aws-typescript-codegen/src/main/java/software/amazon/smithy/aws/typescript/codegen/AwsTraitsUtils.java

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
There's already an `AwsProtocolUtils` where this would fit

Reply:
I think `AwsProtocolUtils` is for protocol specific utilities and this isn't about protocols, so I kept it separate.